pkg/defaults/defaults.go

```diff
@@ -50,6 +50,9 @@ const (
 	DefaultEveRepo           = "https://github.com/lf-edge/eve.git"
 	DefaultRegistry          = "docker.io"
 
+	DefaultEServerTag          = "0.0.2"
+	DefaultEServerContainerRef = "itmoeve/eserver"
```

Comment:
This will need to be one of the lf-edge repos before this could be merged, lets settle on lfedge/eden-http-server (unless you have better alternatives)?

Reply:
Done.

Reply:
@rvs  lfedge/eden-http-server - doesn't work!